### PR TITLE
Add label to highest-rated comment

### PR DESCRIPTION
**Pull request by lubien**
See https://github.com/sindresorhus/refined-github/pull/2108#issuecomment-505378681

image

# Test

leafo/sticky-kit#201
facebook/react#13991
flutter/flutter#11609 has controversial comments
MrMEEE/bumblebee-Old-and-abbandoned#123
microsoft/vscode#71827

### CODE REVIEW

**Comment by lubien** [comment]
// @Lemmmy @bfred-it

File: source/features/highest-rated-comment.tsx [hunk 1]
```diff
@@ -33,7 +33,18 @@ function init(): false | void {
...
 	const {hash} = select<HTMLAnchorElement>('.timestamp', event)!;
 
-	select('.unminimized-comment', event)!.classList.add('rgh-highest-rated-comment');
+	const comment = select('.unminimized-comment', event)!;
+
+	comment.classList.add('rgh-highest-rated-comment');
+
+	select('.timeline-comment-header .timeline-comment-header-text', event)!.parentElement!.lastElementChild!.before((
+		<span
+			className="timeline-comment-label tooltipped"
```
Comment (bfred-it) [inline source/features/highest-rated-comment.tsx]: I think we also need `tooltipped-n`

File: source/features/highest-rated-comment.tsx [hunk 2]
```diff
@@ -33,7 +33,18 @@ function init(): false | void {
 	const avatar = select('.timeline-comment-avatar', event)!.cloneNode(true);
 	const {hash} = select<HTMLAnchorElement>('.timestamp', event)!;
 
-	select('.unminimized-comment', event)!.classList.add('rgh-highest-rated-comment');
+	const comment = select('.unminimized-comment', event)!;
+
+	comment.classList.add('rgh-highest-rated-comment');
+
+	select('.timeline-comment-header .timeline-comment-header-text', event)!.parentElement!.lastElementChild!.before((
```
Comment (bfred-it) [inline source/features/highest-rated-comment.tsx]: Is there a more direct way to add this label?

File: source/features/highest-rated-comment.tsx (hunk 1 above)
Comment (lubien) [reply source/features/highest-rated-comment.tsx]: Done

File: source/features/highest-rated-comment.tsx (hunk 2 above)
Comment (lubien) [reply source/features/highest-rated-comment.tsx]:
```ts
select('.timeline-comment-header-text', event)!.before((
```

Comment (lubien) [reply source/features/highest-rated-comment.tsx]: Also, the reason I put before the `-header-text` is that it's parent sorts in reversed order meaning the header itself being the last child will be the leftmost item.

**Comment by bfred-it** [comment]
Thanks @lubien

**Comment by Lemmmy** [comment]
That's great, much better :smile: thanks @lubien and @bfred-it! Awesome extension.

**Comment by bfred-it** [comment]
Please run this to delete the tags from your fork and locally, we keep pulling them from old forks and pushing some troublesome ones back to this repo

```sh
git tag -l | xargs git tag -d
git ls-remote --tags --refs origin | cut -f2 | xargs git push origin --delete
```

**Comment by lubien** [comment]
> Please run this to delete the tags from your fork and locally, we keep pulling them from old forks and pushing some troublesome ones back to this repo
> 
> ```shell
> git tag -l | xargs git tag -d
> git ls-remote --tags --refs origin | cut -f2 | xargs git push origin --delete
> ```

Done.

**Comment by bfred-it** [comment]
Thank you ^_^

**Pull request merged**